src/main/java/org/jalasoft/moi/model/core/Language.java

```diff
@@ -24,13 +24,16 @@
  */
 public enum Language {
 
-    PYTHON_32(new PythonCommandBuilder(Paths.get("C:\\Users\\Admin\\IdeaProjects\\MOI\\thirdparty\\python\\win" +
+    PYTHON_32(new PythonCommandBuilder(Paths.get(getActualPath() + "\\thirdparty\\python\\win" +
             "\\python32\\Portable_Python_3.2.5.1\\App\\python.exe"), "32")),
     CSHARP(new CsharpCommandBuilder()),
     JAVA(new JavaCommandBuilder()),
     CPP(new CppCommandBuilder());
 
     private final ICommandBuilder commandBuilder;
+    private static String getActualPath(){
+        return System.getProperty("user.dir");
+    }
```

Comment:
I believe it is a good solution, but I would like to put it into an `enum` of constants.

Reply:
Could you specify what object should go into an enum? plase

Reply:
Because we can use it in whatever part of the model, I believe that it's a global constant.